CMakeLists.txt

```diff
@@ -54,7 +65,7 @@ add_library("hh-bbtautau" STATIC ${SOURCE_LIST})
 #set_target_properties("hh-bbtautau" PROPERTIES EXCLUDE_FROM_ALL 1)
 
 foreach(exe_name ${EXE_LIST})
-    target_link_libraries("${exe_name}" hh-bbtautau h-tautau AnalysisTools HTT-utilities SVfit HHKinFit2 TauAnalysisTools TauPOG)
+    target_link_libraries("${exe_name}" hh-bbtautau h-tautau AnalysisTools HTT-utilities SVfit HHKinFit2 TauAnalysisTools TauPOG MulticlassInference)
```

Comment:
MulticlassInference depends on TF and until CMSSW is a "must-have" to use TF, we want to limit dependency on our framework on it. I propose that only CalcMulticlassDNN will have a dependency on MulticlassInference (e.g. can be added on L43 together with TF-related libraries).